packages/core/src/lib/notifire.ts

```diff
@@ -52,12 +52,27 @@ export class Notifire extends EventEmitter {
 
   async registerProvider(
     provider: IEmailProvider | ISmsProvider | IDirectProvider
+  );
+
+  async registerProvider(
+    providerId: string,
+    provider: IEmailProvider | ISmsProvider | IDirectProvider
+  );
+
+  async registerProvider(
+    providerOrProviderId: any,
```

Comment:
I think we need to do a eslint-ginore for this line for the linter to stop complaining about this